File: test/transform/replace/replace_1.c
enum Kind { K1, K2 };

typedef float T;

struct STy {
  struct QTy {
    T V1;
    T V2;
  } * Q;
  unsigned N;
  enum Kind K;
};

void foo(struct STy *S) {
  for (int I = 0; I < S->N; ++I)
    if (S->K == K1) {
      S->Q[I].V1 = S->Q[I].V2 * S->N;
    }
  #pragma spf transform replace(S) nostrict
}
//CHECK: 

File: test/transform/replace_const/replace_1.f
      function foo()
        integer foo
        real a(0:9)
        do i = 1, 10
          a(i - 1) = i
        enddo
        foo = 0
        do 1 i = 0, 8
          foo = foo + a(i)
          a(i + 1) = a(i + 1) + a(i)
1       continue
        foo = foo + a(9)
      end
!CHECK: 
